velox/functions/prestosql/tests/RoundTest.cpp

```diff
@@ -85,6 +85,11 @@ class RoundTest : public functions::test::FunctionBaseTest {
         {11111.0, -4, 10000.0}};
   }
 
+  template <typename T>
+  std::vector<std::tuple<T, int32_t, T>> testRoundWithDecDoubleData() {
+    return {{0.575, 2, 0.58}, {0.574, 2, 0.57}};
+  }
```

Comment:
Can we also add test for nagative values?

Reply:
Just Updated.

Reply:
Would you confirm that this test failed before the fix?